src/style.css

```diff
@@ -0,0 +1,248 @@
+:root {
+  --primary-color: #545862;
+  --secondary-color: #999;
+  --black: #000;
+  --white: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  min-height: 100vh;
+}
+
+body {
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+  font-size: 18px;
+  color: var(--primary-color);
+
+  /* background-color: rgb(171, 206, 209); */
+
+  margin: auto;
+}
```

Comment:
As per CSS best practices you should not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.